packages/authgear-react-native/android/src/main/java/com/authgear/reactnative/AuthgearReactNativeModule.java

```diff
@@ -919,6 +931,88 @@ public void signAnonymousToken(String kid, String data, Promise promise) {
         }
     }
 
+    @ReactMethod
+    public void createDPoPPrivateKey(ReadableMap options, Promise promise) {
+        String kid = options.getString("kid");
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M) {
+            promise.resolve(Arguments.createMap());
+            return;
+        }
+        String alias = this.formatDPoPKeyAlias(kid);
+        try {
+            this.generateKey(alias);
+            promise.resolve("");
```

Comment:
I remember this is a variant of resolve() that allows you return void. Use that instead.

Reply:
There isn't such variant. I can return an empty object instead.